backend/removeReported.js

```diff
@@ -0,0 +1,17 @@
+const getReported = require('./__mock__/getReported');
+ 
+function removeReported(toBeRemoved) {
+    var mockReported = [];
+    mockReported = getReported();
+    if (toBeRemoved == null) { //no email specified
+        return mockReported;
```

Comment:
Codacy has a fix for the issue: [Replace `········` with `····`](https://app.codacy.com/gh/karatwan/CPEN321_world_resources/pullRequest?prid=9811192)
```suggestion
    return mockReported;
```